section4/exercises/tweet.js

```diff
@@ -9,5 +9,36 @@ tweet object instances _behave_ as expected.
 */
 
 class Tweet {
+  constructor(author, content, timeStamp, numberOfLikes, comments) {
+    this.author = author,
+    this.content = content,
+    this.timeStamp = timeStamp,
+    this.numberOfLikes = numberOfLikes,
+    this.comments = comments
+  }
 
-};
\ No newline at end of file
+  addComments(newComment) {
+    this.comments.push(newComment);
+  }
```

Comment:
You'll see this class/function pattern in Mod 1 challenges, where you have a property that is an array and need to write a function that adds more elements to that array. Good job!